openadapt/config.py

```diff
@@ -100,6 +99,14 @@
     list(stop_str) for stop_str in STOP_STRS
 ] + SPECIAL_CHAR_STOP_SEQUENCES
 
+env_file = ".env"
+
+# Create .env file if it doesn't exist
+if not os.path.isfile(env_file):
+    with open(env_file, "w") as f:
+        f.write("OPENAI_API_KEY=<set your api key>\n")
+        f.write("DB_FNAME=openadapt.db\n")
```

Comment:
Please copy .env.example

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/223/commits/3992d1c78d334a352054c2575db248df8677ec4a